README.md

```diff
@@ -72,16 +72,19 @@ Tools can be acquired via instructions in [third_party](/third_party/README.md).
 
 ## Building
 
+The requirements and instructions below are for building the project from source
```

Comment:
The `make generate` target is mentioned below in the `Additional make targets` section. Should that be removed or modified to include the link to contribution guide?

Reply:
I've added a link to the contribution guide